lightning/src/routing/router.rs

```diff
@@ -4106,15 +4107,39 @@ mod tests {
 		{
 			// Attempt to route more than available results in a failure.
 			if let Err(LightningError{err, action: ErrorAction::IgnoreError}) = get_route(
-					&our_id, &payment_params, &network_graph.read_only(), None, 300_000, 42, Arc::clone(&logger), &scorer, &random_seed_bytes) {
-				assert_eq!(err, "Failed to find a sufficient route to the given destination");
+				&our_id, &payment_params, &network_graph.read_only(), None, 300_000, 42,
+				Arc::clone(&logger), &scorer, &random_seed_bytes) {
```

Comment:
Nit: doesn't matter much here and mostly my preference, but by breaking at the paren we can avoid the extra indentation inside the if clause:

```
if let Err(LightningError{err, action: ErrorAction::IgnoreError}) = get_route(
	&our_id, &payment_params, &network_graph.read_only(), None, 300_000, 42,
	Arc::clone(&logger), &scorer, &random_seed_bytes,
) {
	assert_eq!(err, "Failed to find a sufficient route to the given destination");
} else { panic!(); }
```

Reply:
True, this will increase indentation, I however dislike the dangling `) {` and stretching the two lines over four. Can't wait for `rustfmt` though. 😁